api/tests/acceptance/application/answers/answer-controller-save_test.js

```diff
@@ -158,6 +158,8 @@ describe('Acceptance | Controller | answer-controller-save', () => {
         expect(answer.relationships.challenge.data.id).to.equal(answerDB.challengeId);
       });
 
+      // TODO: Fix this the next time the file is edited.
+      // eslint-disable-next-line mocha/no-setup-in-describe
       [
         { locale: FRENCH_FRANCE, expectedCompetenceName: 'Nom de la competence FR' },
         { locale: ENGLISH_SPOKEN, expectedCompetenceName: 'Nom de la competence EN' },
```

Comment:
**question ouverte**
Pour les cas comme ça de boucle `forEach` pour nos tests est-ce que c'est ok de laisser le commentaire de lint ?
Ou est-ce qu'on préconise la duplication des `it` dans ces cas ?
(pour moi ça semble ok de laisser le commentaire de lint pour ces cas là)

Reply:
Je préfère aussi la génération dynamique des tests à la duplication des `it`. Ça me semble ok de laisser l'exception dans ces cas là; on en a relativement peu (ça coûte moins cher de tolérer une exception locale dans de rare cas que de tolérer une approche globale partout).
D'ici la fin du nettoyage, il y aura peut-être une gestion plus fine de ces tests 🤞

Reply:
C'est ajouté dans le dernier commit. Ça me laisse dubitatif de la règle cependant...

D'un côté, on utilise beaucoup de tests dynamiques et ils deviennent plus dur à lire (ils ne sont pas illisibles pour autant) et de l'autre, le setup-in-describe d'éléments en BDD est probablement responsable de nos flaky tests... 

Keep ou drop ?

Reply:
Perso j'attache plus d'importance à la facilité de lecture et de compréhension des tests qu'à la duplication. L'objectif des tests c'est d'exprimer une intention et on fini par la perdre en utilisant les tests dynamiques pour faire de la factorisation de tests car le test devient "générique".
Il y a des cas ou ça reste pratique (pour les tests de validations par exemple), mais on peut toujours faire sans.

Moi je dirais on grade la règle en autorisant les exceptions.